include/rgl/api/core.h

```diff
@@ -151,6 +151,31 @@ static_assert(std::is_trivial<rgl_radar_scope_t>::value);
 static_assert(std::is_standard_layout<rgl_radar_scope_t>::value);
 #endif
 
+/**
+ * Describes 4 bone weights affecting a mesh vertex.
+ * The sum of all weights for a given vertex should equal 1 (RGL do not normalize them).
+ * If a vertex is affected by fewer than 4 bones, each of the remaining weight values must be 0.
+ * bone_idxes for unused bones must still be valid (filled with the existing bone indexes).
+ */
+typedef struct
+{
+	/**
+	 * Array for skinning weights.
+	 * The weight at each index corresponds to the bone_idx with the same index.
+	 */
+	float weights[4];
+	/**
+	 * Array for 4 bone indexes affecting a vertex.
+	 */
+	int32_t bone_idxes[4];
```

Comment:
AFAIK, indexes and indices are both acceptable. I used `idxes` because it is shorter but it doesn't matter here. I'll rename it `bone_indexes` to be consistent with the rest of the code in this PR.